src/components/tx/TxData.js

```diff
@@ -511,7 +537,91 @@ class TxData extends React.Component {
             </div>
           );
         }
-        return undefined;
+        return null;
+      }
+
+      if (!this.props.meta.conflict_with.length) {
+        // it is voided, but there is no conflict
+        return (
+          <div className="alert alert-danger">
+            <h4 className="alert-heading">
+              This {renderBlockOrTransaction()} is voided and <strong>NOT</strong> valid.
+            </h4>
+            <p>
+              This {renderBlockOrTransaction()} is verifying (directly or indirectly) a voided
+              double-spending transaction, hence it is voided as well.
+            </p>
+            <div className="mb-0">
+              <span>
+                This {renderBlockOrTransaction()} is voided because of these transactions:{' '}
+              </span>
+              {renderListWithLinks(this.props.meta.voided_by, true)}
+            </div>
+          </div>
+        );
+      }
+
+      // it is voided, and there is a conflict
+      return (
+        <div className="alert alert-danger">
+          <h4 className="alert-heading">
+            This {renderBlockOrTransaction()} is <strong>NOT</strong> valid.
+          </h4>
+          <div>
+            <span>It is voided by: </span>
+            {renderListWithLinks(this.props.meta.voided_by, true)}
+          </div>
+          <hr />
+          {conflictNotTwin.length > 0 && (
+            <div className="mb-0">
+              <span>Conflicts with: </span>
+              {renderListWithLinks(conflictNotTwin, true)}
+            </div>
+          )}
+          {renderTwins()}
+        </div>
+      );
+    };
+
+    const renderNewUiConflicts = () => {
+      const { twins } = this.props.meta;
+      const conflictNotTwin = this.props.meta.conflict_with.length
+        ? this.props.meta.conflict_with.filter(hash => twins.indexOf(hash) < 0)
+        : [];
+      if (!this.props.meta.voided_by.length) {
+        if (!this.props.meta.conflict_with.length) {
+          // there are conflicts, but it is not voided
+          return (
+            <div className=" alert-success-container">
+              <ValidIcon width="14px" />
+              <span className="alert-success-text">
+                This {renderBlockOrTransaction()} is valid.
+              </span>
+            </div>
+          );
+        }
+
+        if (this.props.meta.conflict_with.length) {
```

Comment:
issue: This case is behaving a bit weird on desktop and actually broken on mobile for the new UI. To test it, use this sample tx ( [link](https://explorer.alpha.nano-testnet.hathor.network/transaction/000000f8ec6a89602a4fb244f6947cff675cf5b6c7510cd0bf323f38bb00e529) ).

I tried searching for a solution for it on the figma, but could not find any. @anafigueired could you lend a hand here?
New UI / Mobile:
![New UI Conflict Winner Tx Mobile](https://github.com/user-attachments/assets/d96b5a41-d68d-4cf9-94e7-fc545620cc2f)

New UI / Desktop:
![New UI Conflict Winner Tx Desktop](https://github.com/user-attachments/assets/0acee429-7ed8-4048-9836-5b36da56cd9c)

Reply:
For reference, here is the transaction that "lost" in the double spending

New UI / Mobile:
![Transaction Conflict Loser Mobile](https://github.com/user-attachments/assets/0d7d69b3-b228-4096-9cc3-dbef1a173bf4)

New UI / Desktop:
![Transaction Conflict Loser Desktop](https://github.com/user-attachments/assets/daf29dcb-dceb-44ce-a990-115d38bf5093)